
zRJ: seems unneeded but in case it comes to haunt us

--- scons/x11.py.orig	2015-10-02 11:45:51.000000000 +0300
+++ scons/x11.py	2015-10-20 13:15:05.000000000 +0300
@@ -43,7 +43,7 @@ def generate(env):
 
 def exists(env):
     # TODO: actually detect the presence of the headers
-    if env['platform'] in ('linux', 'freebsd', 'darwin'):
+    if env['platform'] in ('linux', 'dragonfly', 'freebsd', 'darwin'):
         return True
     else:
         return False
--- scons/gallium.py.orig	2015-10-03 14:30:41.000000000 +0300
+++ scons/gallium.py	2015-10-20 13:16:26.000000000 +0300
@@ -306,7 +306,7 @@ def generate(env):
         cppdefines += ['NDEBUG']
     if env['build'] == 'profile':
         cppdefines += ['PROFILE']
-    if env['platform'] in ('posix', 'linux', 'freebsd', 'darwin'):
+    if env['platform'] in ('posix', 'linux', 'dragonfly', 'freebsd', 'darwin'):
         cppdefines += [
             '_POSIX_SOURCE',
             ('_POSIX_C_SOURCE', '199309L'),
@@ -328,7 +328,7 @@ def generate(env):
                 'GLX_DIRECT_RENDERING',
                 'GLX_INDIRECT_RENDERING',
             ]
-        if env['platform'] in ('linux', 'freebsd'):
+        if env['platform'] in ('linux', 'dragonfly', 'freebsd'):
             cppdefines += ['HAVE_ALIAS']
         else:
             cppdefines += ['GLX_ALIAS_UNSUPPORTED']
@@ -614,7 +614,7 @@ def generate(env):
 
     # Default libs
     libs = []
-    if env['platform'] in ('darwin', 'freebsd', 'linux', 'posix', 'sunos'):
+    if env['platform'] in ('darwin', 'freebsd', 'dragonfly', 'linux', 'posix', 'sunos'):
         libs += ['m', 'pthread', 'dl']
     if env['platform'] in ('linux',):
         libs += ['rt']
